Automattic/wp-calypso
# Classic Editor: remove the Advanced Media Editor experiment

Removes disabled TinyMCE plugin that hasn't been active since 2016 (see also #2085). I ran into it when reviewing usages of `<Dialog>` in #32899.

**How to test:**
Verify that the editor toolbar and image editing in Classic Editor doesn't break after this change.
